docs/references/architecture/inter-process-communication/isolation.md

```diff
@@ -53,7 +53,7 @@ Because the point of the Isolation application is to protect against Development
 
 In this example, we will make a small hello-world style Isolation application and hook it up to an imaginary existing Tauri application. It will do no verification of the messages passing through it, only print the contents to the WebView console.
 
-For the purposes of this example, let's imagine we are in the same directory as `tauri.conf.json`. The existing Tauri application has it's `distDir` set to `../dist`.
+For this example, let's imagine we are in the same directory as `tauri.conf.json`. The existing Tauri application has it's `distDir` set to `../dist`.
```

Comment:
I like the original wording, is there a reason to change it?

Reply:
While I skipped most of the text in the docs as it is not feasibly possible for me to go over it in a single night, I've tried to reduce some sentences by removing needlessly emphasised text and still capturing the original meaning.
If you, however, feel like emphasis here and it is good, then I wholeheartedly welcome the idea!

Reply:
Yeah, I think the extra emphasis here is good.